test/license_test.dart

```diff
@@ -35,7 +35,7 @@ main() {
 
   group('BSD', () {
```

Comment:
Do we have a test for package w/ multiple licenses?

Reply:
I thought I have, but I can't recall it. I remember a license file that has two licenses copied into it.

Reply:
I don't see a test result that exercises it. Would be nice...

Reply:
I'll add this with the next change that will also include the file name in the license object.